cron/centreon-backup.pl

```diff
@@ -312,12 +312,12 @@ ()
 	my @tab_php_ini;
 
 	# PHP CLI
-	my $result = `php -r 'echo php_ini_loaded_file();'`;
+	my $result = `/opt/rh/rh-php71/root/usr/bin/php -r 'echo php_ini_loaded_file();'`;
 	push(@tab_php_ini, trim($result));
 
 	# Apache
-	if ( -e '/etc/php.ini') {
-		push(@tab_php_ini, '/etc/php.ini');
+	if ( -e '/etc/opt/rh/rh-php71/php.ini') {
```

Comment:
What is the purpose of pushing this file ? It seems to be the default file loaded by the PHP binary above.

Reply:
@kduret Do you agree that the additional backup of php.ini is useless ?